blocksuite/affine/widget-drag-handle/src/components/drop-indicator.ts

```diff
@@ -24,13 +24,17 @@ export class DropIndicator extends LitElement {
     if (!this.rect) {
       return null;
     }
+
+    const parentRect = this.parentElement!.getBoundingClientRect();
```

Comment:
Maybe we can remove this `drop-indicator` since we've already have `drag-indicator`?

Reply:
Sure thing. I'll remove it in the upcoming PR.